### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by SUPERCILEX**

### CODE REVIEW

**Comment by SUPERCILEX** [comment]
@JacisNonsense bump

**Comment by JacisNonsense** [comment]
I'm holding off on merging this until I have some time to bundle some more stuff along with this.

File: build.gradle [hunk 1]
```diff
@@ -1,12 +1,13 @@
 plugins {
-    id "jaci.openrio.gradle.GradleRIO" version "2018.01.07" apply false
-    id "jaci.gradle.EmbeddedTools" version "2018.01.06b" apply false
+    id "jaci.openrio.gradle.GradleRIO" version "2018.01.22" apply false
```
Comment (andrewda) [inline build.gradle]: `2018.02.17` is the latest version

**Comment by SUPERCILEX** [comment]
K

**Comment by JacisNonsense** [comment]
Is there a specific reason the deps need to be updated? PF builds just fine with old deps

**Comment by SUPERCILEX** [comment]
Gradle wasn't updated properly so this PR does that. Also, is there a reason _not_ to keep things up-to-date? 😕

**Comment by JacisNonsense** [comment]
It's moreso keeping master up to date with the current release. No need to push an updated version of the library if it's only the build system that gets updated

**Comment by SUPERCILEX** [comment]
There's #35 which makes debugging waaaaaay easier.

**Comment by SUPERCILEX** [comment]
@JacisNonsense Is this good to go?

**Comment by JacisNonsense** [comment]
This won't make any changes to how pathfinder functions - it's stable on the current deps in master. Is there any reason you want to update the build system if nothing's changed? Pathfinder doesn't depend on any other libraries.

**Comment by SUPERCILEX** [comment]
So it doesn't become legacy code?

**Comment by SUPERCILEX** [comment]
@JacisNonsense bump

**Pull request merged**